igor-web/src/main/groovy/com/netflix/spinnaker/igor/jenkins/service/JenkinsService.groovy

```diff
@@ -126,10 +129,12 @@ class JenkinsService implements BuildService{
     }
 
     ScmDetails getGitDetails(String jobName, Integer buildNumber) {
-        new SimpleHystrixCommand<ScmDetails>(
-            groupKey, buildCommandKey("getGitDetails"), {
-            return jenkinsClient.getGitDetails(encode(jobName), buildNumber)
-        }).execute()
+        retrySupport.retry({
+            new SimpleHystrixCommand<ScmDetails>(
+                groupKey, buildCommandKey("getGitDetails"), {
+                return jenkinsClient.getGitDetails(encode(jobName), buildNumber)
+            }).execute()
+        }, maxRetries, retryBackoff, exponentialBackoff)
```

Comment:
`retryBackoff` is in milliseconds, I would probably not do exponential either and just favor more calls in shorter-timeframe vs less calls over longer timeframe (maybe 10 retries, 1-2000ms backoff or something?)

Not a huge amount of value defining the `maxRetries`/`retryBackoff`/`exponentialBackoff` at the top of this class ... they're only used in one spot and not externally configurable.

Reply:
Makes sense! Fixed.